user_notification/tests.py

```diff
@@ -1,7 +1,6 @@
 from user_notification.models import JobAlert
 import pytest
 from datetime import datetime
-from accounts.models import UserAccount
```

Comment:
please squash that commit with the previous one

Reply:
I squashed them, thanks

Reply:
It seems that you still have 3 commits (instead of 2)

Reply:
@Varditli ?

Reply:
I squashed them